common/main/java/com/couchbase/lite/AbstractQuery.java

```diff
@@ -198,15 +194,15 @@ public ListenerToken addChangeListener(@Nullable Executor executor, @NonNull Que
                 (results, err) -> onQueryChanged(token, results, err));
         }
         catch (CouchbaseLiteException e) { throw new IllegalStateException("Failed creating query", e); }
-
         listeners.put(token, queryObserver);
 
         final ExecutionService exec = CouchbaseLiteInternal.getExecutionService();
         exec.postDelayedOnExecutor(
-            UPDATE_DELAY_MS, // 200 ms delay
+            10, // !!! 10 ms delay. work around for CBL-2543. There won't be any delay after CBL-2543 is fixed
```

Comment:
Excellent.  We need a way to guarantee that this gets removed, when the bug is closed.